jobs/delorean/jenkinsfiles/branch/ga/Jenkinsfile

```diff
@@ -61,6 +62,38 @@ GHPullRequest ghFindOrCreatePullRequest(GHRepository repo, String head, String b
     return pr
 }
 
+@NonCPS
+GHPullRequest ghGetPullRequestFromUrl(GitHub gitHub, String prUrl) {
+    URL url = new URL(prUrl)
+    String[] pathSegments = url.path.trim().split('/') - '' - 'pull'
+    String ghOwner = pathSegments[0]
+    String ghRepo = pathSegments[1]
+    String ghPrNumber = pathSegments[2]
+    GHRepository repo = gitHub.getRepository("${ghOwner}/${ghRepo}")
+    return repo.getPullRequest(ghPrNumber as int)
+}
+
+@NonCPS
+GHCommitStatus ghUpdatePrCommitStatus(GHPullRequest pr, GHCommitState state, String targetUrl, String description, String context) {
+    return pr.getRepository().createCommitStatus(pr.getHead().getSha(), state, targetUrl, description, context)
```

Comment:
Do we need to have this in a separate function with the same name? Can we return this on line [83](https://github.com/integr8ly/ci-cd/pull/136/files#diff-19bdd564b8ecac41d9540c5b8ebfa55dR83) instead?

Reply:
We could, but this was copied form an external lib that we will more than likely just end up using, so left it as is.

Reply:
https://github.com/feedhenry/fh-pipeline-library/blob/master/src/org/feedhenry/GitHubUtils.groovy#L89-L97